src/PowerShell/Enums.ts

```diff
@@ -0,0 +1,7 @@
+export  enum AzureCloudName {
```

Comment:
PowerShell folder has code changes related to Azure Powershell login. Sovereign cloud changes should be made in both az cli login and powershell login, hence please move it outside.